tests/testthat/test-parsers.R

```diff
@@ -0,0 +1,25 @@
+test_that("parse_errors_to_df parses correct assertion type and correct error_df", {
```

Comment:
Please make the description more specific. You check the assertion type and if the number of detected errors is correct.

Reply:
Test case & description modified in [33b781a](https://github.com/Appsilon/data.validator/pull/59/commits/33b781a8bec60c537c97cc06b36b88657146e67e).